lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -67,6 +68,7 @@ our @EXPORT = qw(
   sdaf_execute_playbook
   convert_region_to_long
   convert_region_to_short
+  git_clone
```

Comment:
There's no `git_clone` in `sdaf_deployment_library` to export. :)

Reply:
Oh yes. Thanks! 
And fixed.